Tests/MongoSwiftTests/AsyncAwaitTests.swift

```diff
@@ -234,7 +234,7 @@ final class MongoCursorAsyncAwaitTests: MongoSwiftTestCase {
         }
 
         testAsync {
-            let opts = CreateCollectionOptions(capped: true, size: 5)
+            let opts = CreateCollectionOptions(capped: true, max: 5, size: 100_000)
```

Comment:
this is unrelated but this test starting failing against latest once [SERVER-67246](https://jira.mongodb.org/browse/SERVER-67246) went in. `size` is actually supposed to be the max size in bytes, not the number of documents (that's what `max` is for). 
previously the server forced you to have a minimum max size of 4096, so the `size` option here was basically ignored. but that restriction was removed and it is now valid to actually have a 5 byte capped collection. due to this, no documents could actually fit in the collection so this test started to fail.

Reply:
interesting, i'm surprised the previously incorrect value was silently ignored by the server

Reply:
yeah it's interesting; per the docs [here](https://www.mongodb.com/docs/manual/core/capped-collections/#create-a-capped-collection):

> If the size field is less than or equal to 4096, then the collection will have a cap of 4096 bytes.